test/Numerics/Mesh/filter.jl

```diff
@@ -193,13 +202,33 @@ end
                         nothing,
                         dim,
                     )
-                    ClimateMachine.Mesh.Filters.apply!(
-                        Q,
-                        target,
-                        grid,
-                        filter,
-                        direction = direction(),
-                    )
+                    if direction isa EveryDirection
```

Comment:
Applying it in `EveryDirection` now is a bit cumbersome... because we split it into horizontal and vertical filter applications (by basically creating and applying two filters, one for each direction). Not sure if this was the best way, but we don't really know how to handle this case in the filter constructors (what filter matrix do we use?)